### PR TITLE
Adding scripts to be able to do local releases

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: release/gcb/local.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
+
+# shellcheck disable=SC1091
+source gcb_lib.sh
+ROOT=$(cd "$(git rev-parse --show-cdup)" && pwd || return)
+artifacts="$HOME/output/local"
+export NEW_VERSION="local.1"
+export DOCKER_HUB='utka/testing_local'
```
Comment (howardjohn) [inline release/gcb/local.sh]: this probably shouldn't be here?

File: release/gcb/local.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (howardjohn) [inline release/gcb/local.sh]: Why is this under gcb directory? Does it depend on gcb?

File: release/gcb/local.sh (hunk 1 above)
Comment (utka) [reply release/gcb/local.sh]: Haha. oops

File: release/gcb/local.sh (hunk 2 above)
Comment (utka) [reply release/gcb/local.sh]: Because the rest of the scripts are here, that's the only reason.

**Comment by utka** [comment]
Added README.md

**Comment by howardjohn** [comment]
@linsun fyi

**Pull request merged**